tasks/install.yml

```
@@ -30,17 +30,32 @@
     state: directory
     mode: 0755
 
-- name: Create config dir
+- name: Create config directories
   file:
     path: "{{ item }}"
     state: directory
     owner: root
     group: "{{ promtail_system_group }}"
     mode: 0770
-  with_items:
+  loop:
     - "{{ promtail_config_dir }}"
     - "{{ promtail_config_file_sd_dir }}"
 
+- name: Check whether the data exists
+  stat:
+    path: "{{ promtail_positions_path | dirname }}"
+  register: promtail_positions_folder_stat
+  changed_when: False
+
+- name: Create data directory
+  file:
+    path: "{{ promtail_positions_path | dirname }}"
+    state: directory
+    owner: root
+    group: "{{ promtail_system_group }}"
+    mode: ug=rwx,o=
```

```suggestion
    mode: 0770
```

Let's please use the same syntax for all tasks. I am open to adjusting all tasks with a different syntax if there is any benefit to it 👍